assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -100,6 +100,17 @@ function useCanvasKeys(ref) {
 
     const doc = container.ownerDocument;
 
+    const isDescendant = (parent, child) => {
```

Comment:
Could we use [Node.contains](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains) here?

Reply:
Use the anchorOffset approach, no longer need this helper function.